src/app/weather-wind/weather-wind.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-weather-wind',
+  template: '<p> Wind : {{weatherWind}} </p>',
+  styleUrls: ['./weather-wind.component.scss']
```

Comment:
We should also show a icon so that the user knows that we are talking about the wind. 
For now we explicitly say "Wind" but I think that we might not be able to fit all text. 
image.png
```suggestion
  styleUrls: ['./weather-wind.component.scss']
```

Reply:
I added you can check